zipkin-cassandra/src/schema/cassandra-schema-cql3.txt

```diff
@@ -0,0 +1,73 @@
+CREATE KEYSPACE "Zipkin" WITH replication = {'class': 'NetworkTopologyStrategy', 'us-west': '1'}  AND durable_writes = true;
```

Comment:
So how about we say that unless there are some resource constraints lets just try to do 3?